src/FsCheck/Reflect.fs

```diff
@@ -60,13 +60,15 @@ module internal Reflect =
         ctor.Invoke
 
     /// Returns the case name, type, and functions that will construct a constructor and a reader of a union type respectively
-    let getUnionCases unionType : (string * (int * System.Type list * (obj[] -> obj) * (obj -> obj[]))) list = 
+    let getUnionCases : Type -> (string * (int * System.Type list * (obj[] -> obj) * (obj -> obj[]))) list =
```

Comment:
Possible a breaking ABI change. I will inspect the generated binaries to if it changes the signature. If so changing the signature back to previous one isn't a big deal.